packages/mobile/src/screens/collection-screen/CollectionScreen.tsx

```diff
@@ -151,10 +149,6 @@ const CollectionScreenComponent = (props: CollectionScreenComponentProps) => {
     created_at,
     is_delete
   } = collection
-  const isOfflineModeEnabled = useIsOfflineModeEnabled()
```

Comment:
did we mean to remove this one?

Reply:
it's also on the delete list. just sliding it in there